File: Uri/1012.py

x = raw_input()
z = x.split(' ')
a = float(z[0])
b = float(z[1])
c = float(z[2])

tri = a*c/2
cir = 3.14159*c*c;
tra = (a+b)*c/2.0;
qua = b*b;
ret = a*b;

print "TRIANGULO: %.3f"%tri
print "CIRCULO: %.3f"%cir
print "TRAPEZIO: %.3f"%tra
print "QUADRADO: %.3f"%qua
print "RETANGULO: %.3f"%ret

File: Uri/1012.c
#include<stdio.h>
#include<stdlib.h>

int main(){
    double a, b, c, t, x, tp, q, r, pi;
    pi = 3.14159;
    scanf("%lf %lf %lf", &a, &b, &c);
    t = (a * c)/2;
    x = pi * c * c;
    tp = ((a + b) * c)/2;
    q = b * b;
    r = a * b;
    printf("TRIANGULO: %.3lf\n", t);
    printf("CIRCULO: %.3lf\n", x);
    printf("TRAPEZIO: %.3lf\n", tp);
    printf("QUADRADO: %.3lf\n", q);
    printf("RETANGULO: %.3lf\n", r);
    return 0;
}
